### ISSUE TITLE
Bug of UV coordinate

### ISSUE THREAD

**Issue by caiyongqi** [sxyu/pixel-nerf#48]
In models.py, `uv = -xyz[:, :, :2] / xyz[:, :, 2:]`, which causes its origin to be in the  left bottom corner of the image, since your R will convert the world coordinate system to the camera coordinate system in OPGL format.
Maybe the y-axis (v) needs to be flipped, i.e. `uv = uv * torch.tensor([[[1, -1]]], device=uv.device)`.
